packages/gatsby-recipes/src/providers/gatsby/site-metadata.js

```diff
@@ -155,14 +163,20 @@ class BabelPluginSetSiteMetadataField {
 
             const newSiteMetadata = newSiteMetadataTemplate.declarations[0].init
 
-            right.properties = right.properties.map(p => {
-              if (p.key.name !== `siteMetadata`) return p
-
-              return {
-                ...p,
-                value: newSiteMetadata,
-              }
-            })
+            if (siteMetadataExists) {
+              right.properties = right.properties.map(p => {
+                if (p.key.name !== `siteMetadata`) return p
+
+                return {
+                  ...p,
+                  value: newSiteMetadata,
+                }
+              })
+            } else {
+              right.properties.unshift(
```

Comment:
@mxstbr was fixing this parallelism support & also added blank slate support i.e. when gatsby-config.js doesn't have `siteMetadata` added.

Reply:
:100: This is great, thank you!!